test/recipes/82-test_tfo_cli.t

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/env perl
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use warnings;
+
+use IPC::Open2;
+use OpenSSL::Test qw/:DEFAULT srctop_file bldtop_file/;
+use OpenSSL::Test::Utils;
+
+setup("test_tfo");
+
+plan skip_all => "test_tfo_cli needs sock enabled" if disabled("sock");
+plan skip_all => "test_tfo_cli needs tls enabled" if disabled("tls");
```

Comment:
It seems to need a TLS version <1.3 enabled

Reply:
Is `tls` not sufficient for this, then? Not sure what you're asking for here?

Reply:
"tls" checks if *a* TLS version is enabled. In the event that *only* TLSv1.3 is enabled then the test would not be skipped, and would then fail.

Reply:
Fixed